.github/workflows/main.yml

```diff
@@ -35,15 +45,15 @@ jobs:
 
     # Run data ----
     - name: Run report data
-      if: ${{ github.ref == 'refs/heads/main' }}
+      if: ${{ github.ref == 'refs/heads/main' || github.event_name == 'workflow_dispatch'}}
       working-directory: reports
       run: |
         make parameters
         make data
 
     # copy data ----
     - name: Copy development report data
-      if: ${{ github.ref == 'refs/heads/main' }}
+      if: ${{ github.ref == 'refs/heads/main' || github.event_name == 'workflow_dispatch'}}
       run: |
         gsutil -m rsync -r reports/outputs/ gs://test-calitp-reports-data/report_gtfs_schedule/
     - name: Copy production report data
```

Comment:
Does this one also need to be enabled in a specific workflow dispatch event?

Reply:
Good catch, I think so actually. Just updated this one and the two next to it.